Sources/Protocols/Animatable/Animatable.swift

```diff
@@ -499,6 +501,48 @@ fileprivate extension Animatable where Self: UIView {
     }
   }
 
+  func scale(fromX: Double, fromY: Double, toX: Double, toY: Double, configuration: AnimationConfiguration, completion: AnimatableCompletion? = nil) {
+    if fromX.isNaN || fromY.isNaN || toX.isNaN || toY.isNaN {
+      return
+    }
+    if case .none = configuration.timingFunction {
```

Comment:
`scaleWithTimingFunction`  / `scaleWithoutTimingFunction` 😬 
(not really fan, but can't come with a better name right now)